p2p/log_diff_test.go

```diff
@@ -1,73 +1,71 @@
 package p2p
 
 import (
+	"context"
 	"testing"
+
+	"github.com/qri-io/cafs"
+	"github.com/qri-io/dataset/dstest"
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/base"
+	p2ptest "github.com/qri-io/qri/p2p/test"
 )
 
 func TestRequestLogDiff(t *testing.T) {
-	// ctx := context.Background()
-	// streams := ioes.NewDiscardIOStreams()
-	// factory := p2ptest.NewTestNodeFactory(NewTestableQriNode)
-	// testPeers, err := p2ptest.NewTestDirNetwork(ctx, factory)
-	// if err != nil {
-	// 	t.Fatalf("error creating network: %s", err.Error())
-	// }
-	// if err := p2ptest.ConnectQriNodes(ctx, testPeers); err != nil {
-	// 	t.Fatalf("error connecting peers: %s", err.Error())
-	// }
+	ctx := context.Background()
+	streams := ioes.NewDiscardIOStreams()
+	factory := p2ptest.NewTestNodeFactory(NewTestableQriNode)
+	testPeers, err := p2ptest.NewTestDirNetwork(ctx, factory)
+	if err != nil {
+		t.Fatalf("error creating network: %s", err.Error())
+	}
+	if err := p2ptest.ConnectQriNodes(ctx, testPeers); err != nil {
+		t.Fatalf("error connecting peers: %s", err.Error())
+	}
 
-	// peers := asQriNodes(testPeers)
+	peers := asQriNodes(testPeers)
 
-	// tc, err := dstest.NewTestCaseFromDir("testdata/tim/craigslist")
-	// if err != nil {
-	// 	t.Fatal(err)
-	// }
+	tc, err := dstest.NewTestCaseFromDir("testdata/tim/craigslist")
+	if err != nil {
+		t.Fatal(err)
+	}
 
-	// // add a dataset to tim
-	// ref, _, err := base.CreateDataset(peers[4].Repo, streams, tc.Name, tc.Input, tc.BodyFile(), false, true)
-	// if err != nil {
-	// 	t.Fatal(err)
-	// }
+	// add a dataset to peer 5
```

Comment:
Don't count starting-from-1 for the comments if the code is using starting-from-0. Change this to `// add a dataset to peer 4 (counting starting at 0)`.